public/App.js

```diff
@@ -1,5 +1,5 @@
 // Insert your local IP + port here
-const socket = new WebSocket('ws://192.168.0.104:7080');
+const socket = new WebSocket('ws://192.168.0.114:7080');
```

Comment:
Replacing _my_ hardcoded IP with yours?! Unacceptable

Reply:
I was kidding by the way this can stay. Obviously this isn't ideal so maybe if app & game server are always guaranteed to be the same host, this IP is just 127.0.0.1. Not sure if game & app server will always be the same though.